arkouda/pdarraycreation.py

```diff
@@ -653,6 +655,8 @@ def standard_normal(size : int, seed : Union[None, int]=None) -> pdarray:
     ----------
     size : int
         The number of samples to draw (size of the returned array)
+    seed : int
+        Index for where to pull the first returned value
```

Comment:
"Value used to initialize random number generator"

Reply:
updated